src/JJMasterData.Core/JJMasterData.Core.csproj

```diff
@@ -41,6 +42,16 @@
   <ItemGroup>
     <ProjectReference Include="..\JJMasterData.Commons\JJMasterData.Commons.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="DI\**" />
+    <Compile Remove="DataDictionary\Repository\DataDictionaryFactory.cs" />
+    <Compile Remove="FormEvents\FormEventResolverFactory.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Remove="DI\**" />
+  </ItemGroup>
```

Comment:
Oops

Reply:
I think you remove it from the project instead of deleting, that's why I didn't saw you removed DataDictionaryFactory.